src/data/markdown/docs/30 xk6-browser/01 xk6-browser/02 BrowserContext.md

```diff
@@ -3,33 +3,24 @@ title: "BrowserContext"
+
+| Method                                                                                                                                          | Description                                                                                                |
+|-------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| [BrowserContext.browser()](/javascript-api/xk6-browser/browsercontext/browser/)                                                                 | Returns the [Browser](/javascript-api/xk6-browser/browser) instance that this `BrowserContext` belongs to. |
+| <BWIPT id="442"/> [BrowserContext.clearCookies()](/javascript-api/xk6-browser/browsercontext/clearcookies/)                                     | Clears the `BrowserContext`'s cookies.                                                                     |
+| <BWIPT id="443"/> [BrowserContext.clearPermissions()](/javascript-api/xk6-browser/browsercontext/clearpermissions)                              | Clears all permission overrides for the `BrowserContext`.                                                  |
+| [BrowserContext.close()](/javascript-api/xk6-browser/browsercontext/close)                                                                      | Close the `BrowserContext` and all its [page](/javascript-api/xk6-browser/page)s.                          |
+| [BrowserContext.grantPermissions(permissions[, options])](/javascript-api/xk6-browser/browsercontext/grantpermissions)                          | Grants specified permissions to the `BrowserContext`.                                                      |
+| [BrowserContext.newPage()](/javascript-api/xk6-browser/browsercontext/newpage)                                                                  | Creates a new [page](/javascript-api/xk6-browser/page) inside this `BrowserContext`.                       |
+| <BWIPT id="444"/> [BrowserContext.pages()](/javascript-api/xk6-browser/browsercontext/pages)                                                    | Returns a list of [page](/javascript-api/xk6-browser/page)s inside this `BrowserContext`.                  |
+| <BWIPT id="445"/> [BrowserContext.setDefaultNavigationTimeout(timeout)](/javascript-api/xk6-browser/browsercontext/setdefaultnavigationtimeout) | Sets the default navigation timeout in milliseconds.                                                       |
+| <BWIPT id="456"/> [BrowserContext.setDefaultTimeout(timeout)](/javascript-api/xk6-browser/browsercontext/setdefaulttiontimeout)                                   | Sets the default maximum timeout in milliseconds.                                                          |
```

Comment:
Should we use the one in the inner documentation?

"Sets the default maximum timeout for all methods accepting a timeout option."

Reply:
Resolved by: 5330622de31601fb8e2d585206567d1214500d37